tests/forcefields/conftest.py

```diff
@@ -13,3 +16,16 @@ def pytest_runtest_setup(item: Any) -> None:
     torch.set_default_dtype(torch.float32)
     # For consistent performance across hardware, explicitly set device to CPU
     torch.set_default_device("cpu")
+
+
+@pytest.fixture(scope="session", autouse=True)
+def download_deepmd_pretrained_model(test_dir: Path) -> None:
+    # Download DeepMD pretrained model from GitHub
+    file_url = "https://raw.github.com/sliutheorygroup/UniPero/main/model/graph.pb"
+    local_path = test_dir / "forcefields" / "deepmd" / "graph.pb"
+    response = requests.get(file_url, timeout=10)
```

Comment:
`atomate2` doesn't currently depend on `requests` so not sure it's installed. if it is, only as transitive dep. maybe better to use std lib `urllib.request.urlretrieve`